images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -52,145 +59,144 @@ function Get-RustClippyVersion {
-    return "Node $nodeVersion"
+    return $nodeVersion
 }
 
 function Get-ChocoVersion {
-    return "Chocolatey $(choco --version)"
+    return $(choco --version)
 }
 
 function Get-VcpkgVersion {
     $commitId = git -C "C:\vcpkg" rev-parse --short HEAD
-    return "Vcpkg (build from commit $commitId)"
+    return "(build from commit <$commitId>)"
 }
 
 function Get-NPMVersion {
-    return "NPM $(npm -version)"
+    return $(npm -version)
 }
 
 function Get-YarnVersion {
-    return "Yarn $(yarn -version)"
+    return $(yarn -version)
 }
 
 function Get-RubyGemsVersion {
-    return "RubyGems $(gem --version)"
+    return $(gem --version)
 }
 
 function Get-HelmVersion {
     ($(helm version --short) | Out-String) -match "v(?<version>\d+\.\d+\.\d+)" | Out-Null
     $helmVersion = $Matches.Version
-    return "Helm $helmVersion"
+    return $helmVersion
 }
 
 function Get-PipVersion {
     ($(pip --version) | Out-String) -match "(?<version>pip [\d\.]+) .+ (?<python>\(python [\d\.]+\))" | Out-Null
     $pipVersion = $Matches.Version
     $pythonVersion = $Matches.Python
-    return "$pipVersion $pythonVersion"
+    return ("$pipVersion $pythonVersion" -replace "^pip").Trim()
 }
 
 function Get-CondaVersion {
-    $condaVersion = & "$env:CONDA\Scripts\conda.exe" --version
-    return "Mini$condaVersion (pre-installed on the image but not added to PATH)"
+    $condaVersion = ((& "$env:CONDA\Scripts\conda.exe" --version) -replace "^conda").Trim()
+    return "$condaVersion (pre-installed on the image but not added to PATH)"
 }
 
 function Get-ComposerVersion {
-    composer --version | Take-Part -Part 0,2
+    composer --version | Take-Part -Part 2
 }
 
 function Get-NugetVersion {
-    (nuget help) -match "NuGet Version" -replace "Version: "
+    return (((nuget help) -match "NuGet Version") -replace "NuGet Version:").Trim()
 }
 
 function Get-AntVersion {
     ($(ant -version) | Out-String) -match "version (?<version>\d+\.\d+\.\d+)" | Out-Null
     $antVersion = $Matches.Version
-    return "Ant $antVersion"
+    return $antVersion
 }
 
 function Get-MavenVersion {
     ($(mvn -version) | Out-String) -match "Apache Maven (?<version>\d+\.\d+\.\d+)" | Out-Null
     $mavenVersion = $Matches.Version
-    return "Maven $mavenVersion"
+    return $mavenVersion
 }
 
 function Get-GradleVersion {
     ($(gradle -version) | Out-String) -match "Gradle (?<version>\d+\.\d+)" | Out-Null
     $gradleVersion = $Matches.Version
-    return "Gradle $gradleVersion"
+    return $gradleVersion
 }
 
 function Get-SbtVersion {
-    $sbtVersion = (sbt -version) -match "sbt script version:" -replace "script version: "
-    return "$sbtVersion"
+    return ((sbt -version) -match "sbt script version:" -replace "sbt script version:").Trim()
 }
 
 function Get-DotnetSdks {
     $sdksRawList = dotnet --list-sdks
-    $sdkVersions = ($sdksRawList | Foreach-Object {$_.Split()[0]}) -join ', '
+    $sdkVersions = ($sdksRawList | Foreach-Object {$_.Split()[0]}) -join ' '
```

Comment:
I think we don't need to make `join` at all. We need `$sdkVersions` to be the array of versions instead of the single string.

Reply:
thanks for the catch - corrected, was my fault, somehow I overlooked it